TestnetGuide/README.md

```diff
@@ -0,0 +1,217 @@
+# Althea Testnet guide
+
+![Althea_Logo-BLUE_SIGNAL](https://user-images.githubusercontent.com/44331529/218240936-c2095305-1a28-45f6-8ccd-d068a4fe5754.svg)
+
+[WebSite](https://www.althea.net/)\
+[GitHub](https://github.com/althea-net)
+=
+[EXPLORER](https://explorer.stavr.tech/althea-testnet/staking)
+=
+
+- **Minimum hardware requirements**:
+
+| Node Type |CPU | RAM  | Storage  | 
+|-----------|----|------|----------|
+| Testnet   |   4|  8GB | 150GB    |
+
+
+# 1) Auto_install script
+```python
+wget -O althe https://raw.githubusercontent.com/obajay/nodes-Guides/main/Althea/althe && chmod +x althe && ./althe
```

Comment:
looks like this is mispelled?

also if we're going to merge a guide that grabs a python file I think moving that file into this repo and then having this auto install script as part of hte main guide (not just another file) would be best.

Reply:
> looks like this is mispelled?
> 
> also if we're going to merge a guide that grabs a python file I think moving that file into this repo and then having this auto install script as part of hte main guide (not just another file) would be best.

1) the path has been changed

wget -O althe https://raw.githubusercontent.com/obajay/nodes-Guides/main/Projects/Althea/althe && chmod +x althe && ./althe

2) and it's not a python. this is bash script

and the fact that python is written - there is just a stylistic form.

```python
12EXEMPLE
```
```bash
12EXEMPLE
```
```java
12EXEMPLE
```
```html
12EXEMPLE
```